src/components/renderer/form-nested-screen.vue

```diff
@@ -54,21 +54,41 @@ export default {
     },
   },
   methods: {
+    elementsValid(config) {
```

Comment:
The name `elementsValid` doesn't really communicate anything to me. Since this function is hiding all submit buttons, I think a name like `hideSubmitButtons` would communicate intent much better.

Reply:
applied change